extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/app.js

```diff
@@ -52,6 +52,15 @@ angular.module('PaperUI', [ 'PaperUI.controllers', 'PaperUI.controllers.control'
         templateUrl : 'partials/configuration.html',
         controller : 'ConfigurationPageController',
         title : 'Configuration'
+    }).when('/configuration/items', {
+        templateUrl : 'partials/configuration.html',
+        controller : 'ConfigurationPageController',
+        title : 'Configuration'
+    }).when('/configuration/item/edit/:itemName', {
+        templateUrl : 'partials/item.config.html'
```

Comment:
Actually title is unnecessary in some other routes as well as we are dynamically setting it anyway in the controller. e.g. thing/edit route sets title to 'Configuration' but it is removed as we set the path to thing UID inside its controller.